### PR TITLE
Initial commit for tabs with icon and svg

### LINKED ISSUE #2: Tabs not responsive
@K-Kumar-01 to make tabs responsive you can look here to https://www.bbc.com/ .

### PR DESCRIPTION
**Pull request by K-Kumar-01**
A basic implementation of tabs with icon and svg.
Tell about the basic design and the future implementations and needs

### CODE REVIEW

**Comment by king-11** [comment]
A Few Recommendations
- Look into transitions because the SVG popping out quickly doesn't look great.
- How abt aligning text below SVG and on a smaller window adding media query to display
  SVG and no text.

Add the above stuff, further if you want I can look into the sliding tab that's currently static after you are done.

File: Implementation/Tabs with icon and svg/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <img class = "swapper-img" src="icons/chat.png" alt="">
+      </li>
+      <li class="swp swappper-item">
+        Videos
+        <img class = "swapper-img" src="icons/video.png" alt="">
+      </li>
+      <li class="swp swappper-item">
+        Photos
+        <img class = "swapper-img" src="icons/photos.png" alt="">
+      </li>
```
Comment (king-11) [inline Implementation/Tabs with icon and svg/index.html]: Include them in a ul tag and for a more clear structure include Text inside anchor tag of each li and add proper classes.

File: Implementation/Tabs with icon and svg/javascript/master.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+swapItems.forEach((item, i) => {
+  item.addEventListener('mouseover', () => {
+    item.children[0].style.display="block";
+  });
+  item.addEventListener('mouseout', () => {
+    item.children[0].style.display="none";
+  });
+  item.addEventListener('click',() => {
+    document.querySelector('.swp-active').classList.remove("swp-active");
+    item.classList.add("swp-active");
```
Comment (king-11) [inline Implementation/Tabs with icon and svg/javascript/master.js:5]: This looks good but more aesthetic feel would be by making it scroll

Comment (K-Kumar-01) [reply Implementation/Tabs with icon and svg/javascript/master.js:5]: Could you please explain it. A gif would help a lot.
I didnt quite understand it.

File: Implementation/Tabs with icon and svg/index.html (hunk 1 above)
Comment (K-Kumar-01) [reply Implementation/Tabs with icon and svg/index.html]: I have added the ul tag along with alt text and title. also images now appear more smotthly

Comment (king-11) [reply Implementation/Tabs with icon and svg/index.html]: LGTM!

File: Implementation/Tabs with icon and svg/javascript/master.js (hunk 2 above)
Comment (king-11) [reply Implementation/Tabs with icon and svg/javascript/master.js:5]: Whenever we change the active tab it should move its way throught the page.

**Pull request merged**